ci-operator/jobs/infra-periodics.yaml

```diff
@@ -119,6 +119,52 @@ periodics:
         requests:
           cpu: 500m
     serviceAccountName: upgrader
+- agent: kubernetes
+  cluster: app.ci
+  cron: 0 5 * * 1-5
+  decorate: true
+  labels:
+    ci.openshift.io/role: infra
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-hosted-mgmt-hypershift-deployment
+  reporter_config:
+    slack:
+      channel: '#ops-testplatform'
+      job_states_to_report:
+      - failure
+      report_template: Job {{.Spec.Job}} failed which indicates that the hypershift
+        deployment on `hosted-mgmt` has not been completed. Please check if anything
+        blocks the process.
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        export MGMT_AWS_CONFIG_PATH='/tmp/fake.aws'
+        echo -e "[default]\naws_access_key_id=AKKKKKKKKKKKKKKKKKKK\naws_secret_access_key=Hasdasdsadasdadsadsadsadk" > "$MGMT_AWS_CONFIG_PATH"
```

Comment:
You don't need to do that on the fly. You can just mount a secret in that path directly.

Reply:
this is a fake credential